pyproject.toml

```diff
@@ -2,16 +2,44 @@
 line-length = 100
 target-version = ["py38", "py39", "py310"]
 
+
 [tool.isort]
 profile = "black"
 py_version = 38
 line_length = 100
 combine_as_imports = true
 filter_files = true
 
+
+[tool.taskipy.settings]
+runner = "dotenv -f task.env run --"
```

Comment:
we can use dotenv in the workflow in order to set this variable so there's only one place where there's a constant.